docs-ref-conceptual/authenticate-azure-cli-interactively.md

```diff
@@ -45,6 +45,29 @@ az login --user <username> --password <password>
 > az login -u $AzCred.UserName -p $AzCred.GetNetworkCredential().Password
 > ```
 
+## Refresh tokens
+
+When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. When a access token is granted, it is only valid for a short amount of time. Therefore, a refresh token is also generated at the same time, so that the client application can exchange this refresh token for a new access token when needed. 
```

Comment:
```suggestion
When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. When an access token is granted, it is only valid for a short amount of time. Therefore, a refresh token is generated at the same time, so that the client application can exchange an expired token for a new access token when needed. 
```